metrics/cli/lib/gcloud/adapter/gcloud_cli_service_adapter.dart

```diff
@@ -40,7 +40,8 @@ class GCloudCliServiceAdapter implements GCloudService {
     await _gcloudCli.createProject(projectId);
 
     await _gcloudCli.listRegions(projectId);
-    final region = _prompter.prompt(GCloudStrings.enterRegionName);
+    final regionPrompt = _prompter.prompt(GCloudStrings.enterRegionName);
+    final region = _formatRegionName(regionPrompt);
```

Comment:
I believe it would be better to just trim the entered region name. 
```suggestion
    final region = regionPrompt.trim();
```